src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -661,6 +661,24 @@ def build_load_balancer_resource(cmd, name, location, tags, backend_pool_name, n
     }
     if sku and cmd.supported_api_version(ResourceType.MGMT_NETWORK, min_api='2017-08-01'):
         lb['sku'] = {'name': sku}
+        # LB rule is the way to enable SNAT so outbound connections are possible
+        if sku.lower() == 'standard':
```

Comment:
We need to re-record (or at least re-run live) the network tests for LB to ensure that this change doesn't inadvertently break them.  It would not be reflected in simply playing back the recordings.